src/corecel/data/Collection.hh

```diff
@@ -132,8 +132,8 @@ using ItemRange = Range<OpaqueId<T, Size>>;
 template<class T1, class T2>
 class ItemMap
 {
-    static_assert(detail::IsOpaqueId<T1>::value, "T1 is not OpaqueID");
-    static_assert(detail::IsOpaqueId<T2>::value, "T2 is not OpaqueID");
+    static_assert(detail::is_opaque_id_v<T1>, "T1 is not OpaqueID");
+    static_assert(detail::is_opaque_id_v<T2>, "T2 is not OpaqueID");
```

Comment:
We should be more consistent on the naming scheme; I think sticking with our internal class conventions rather than making it look like `std` is better, since we have `Span`? (But looks like we might want to rename `celeritas::numeric_limits` 🤪 )

Reply:
I was looking through the codebase and most constexpr bool are using snake case (in ApplierTraits, ActionRegistryIMpl)

Reply:
maybe we can update this in a separate PR?

Reply:
Works for me; added to #721.